apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -205,7 +205,7 @@ export class SendMessagePush extends SendMessageBase {
 
       const noDeviceTokensAndNoOverrides = !deviceTokens && !uniqueOverrideChannels?.length;
       // We avoid to send a message if subscriber has not an integration or if the subscriber has no device tokens for said integration
-      if (noDeviceTokensAndNoOverrides || !integration || isChannelMissingDeviceTokens) {
+      if (noDeviceTokensAndNoOverrides || !integration) {
         continue;
```

Comment:
@LetItRock this was breaking delivery with fcm when only `topic` was used, as per our docs: https://docs.novu.co/platform/integrations/push/fcm#sending-notifications-to-fcm-topics

Just wanted to check with you if we haven't missed something.